app/controllers/concerns/vendor_api/application_data_concerns.rb

```diff
@@ -27,7 +27,7 @@ def application_choices_visible_to_provider(includes = include_properties)
     end
 
     def exclude_deferrals
-      full_api_version_number.eql?(VendorAPI::PRELIMINARY_VERSION)
+      full_api_version_number.eql?(VendorAPI::VERSION_EXCLUDING_DEFERRED_OFFERS)
```

Comment:
Im going to be using this version to lock validating against the schema, is it possible to name it something more generic, as initial version etc.?

Reply:
Hmm, struggling a little with this. `VERSION_1_0` is really what's meant here and I'm only half-joking :)